### DISCUSSION TITLE
[Support]: What is this timestamp I see in the left when I set Crop to false in snapshots?

### DISCUSSION THREAD

**Discussion by atv2016** [blakeblackshear/frigate#10492] [General Support]
### Describe the problem you are having

When I disable crop all of a sudden I am seeing what looks like an additional timestamp ? Is that a frigate timestamp?
Added a screenshot of this behaviour.

### Version

na

### Frigate config file

```yaml
na
```

### Relevant log output

```shell
na
```

### FFprobe output from your camera

```shell
na
```

### Frigate stats

```json
na
```

### Operating system

Other

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

sans

**Comment by atv2016** [comment]
![Screenshot 2024-03-16 at 10 23 32](https://github.com/blakeblackshear/frigate/assets/16917203/67c367d1-b5ab-4360-9d52-166b547d7b96)

**Reply by NickM-27** [reply]
frigate doesn't add a timestamp to the snapshot unless it is configured to

**Comment by dz4news** [comment]
That is most likely coming from your camera as an overlay.
I have dahua cameras and if I dont disable the timestamp setting on the camera itself, then I get a similar issue.
I would log directly into your camera(s) settings and adjust accordingly.
